src/views/name.njk

```diff
@@ -3,17 +3,94 @@
 {% block content %}
 
 {% if props.parameters['which-type'] == "registerLp" %}
-   {% include "pages/name/limited-partnership-name.njk" %}
+  {% set title = i18n.namePage.title %}
```

Comment:
to avoid using props.parameters['which-type'] each time, can you create another var , something like partnershipType, and use that in the if statements in place props.parameters['which-type']

Reply:
Done.